### PR TITLE
Add script for updating attendances.

### PR DESCRIPTION
**Pull request by godwinpang**
To whoever is reading this - sorry :)

I'm too lazy to write everything out, so tl;dr:

1. Grab the check in csv (export gsheets as csv) and run the python script - it'll generate a json
2. The `restore_points.ts` script does a bunch of stuff
   - Restores points for people who were checked off but did not get points
   - Fix up a couple events (13/33) that couldn't be checked off
   - Adds in people from the event check in Google Sheets

Run the script like this `npm run build; NODE_ENV=development node -r dotenv/config -r module-alias/register dist/scripts/restore_points.js`

The script will run against whatever db is specified in your `.env` file.

### CODE REVIEW

File: src/scripts/parse_attendance.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import json
+
+with open('attendance.csv') as file:
+    lines = file.readlines()
+    cleanLines = [line.strip().split(',') for line in lines][1:]
+
+    # email, name, id, hours
+    idx = [1,-5,-4,-3]
```
Comment (thaigillespie) [inline src/scripts/parse_attendance.py:8]: lmao holy, why tho

File: src/scripts/restore_points.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+    await attendanceRepo.save(attendance);
+  }
+
+  // do stuff from g sheets
+  fs.readFile('./src/scripts/attendance.json', 'utf8', async (_, data) => {
+    const rawAttendances = JSON.parse(data).attendances;
+    for (const a of rawAttendances) {
+      const user = await userRepo.findOne({ where: { email: a.email } });
+      const officer = await userRepo.findOne({ where: { id: 16 } });
```
Comment (thaigillespie) [inline src/scripts/restore_points.ts:40]: Why this specific officer?

File: src/scripts/restore_points.ts [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  const attendances = await attendanceRepo.find({ relations: ['attendee', 'event'] });
+
+  // calculate attendance points
+  for (const attendance of attendances) {
+    if (!attendance.endTime && attendance.event.id != 13) {
+      continue;
+    }
+
+    // these two events need to be fixed up
+    if (attendance.event.id == 13 || (attendance.event.id == 33 && attendance.isInductee)) {
```
Comment (thaigillespie) [inline src/scripts/restore_points.ts:21]: What happened to 13 and 33 lol

**Review by thaigillespie — APPROVED** [review]
TLDR lgtm

Since this is just a one-off script (hopefully), not much to say here besides some questions. As long as it does what it needs to then we're good.

File: src/scripts/parse_attendance.py (hunk 1 above)
Comment (godwinpang) [reply src/scripts/parse_attendance.py:8]: :)

File: src/scripts/restore_points.ts (hunk 3 above)
Comment (godwinpang) [reply src/scripts/restore_points.ts:21]: :)
